### DISCUSSION TITLE
Welcome to UserFrosting Discussions!

### DISCUSSION THREAD

**Discussion by lcharette** [userfrosting/UserFrosting#1145] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Silic0nS0ldier** [comment]
Well, might as well participate.

I'm Jordan, a frontend software engineer at Canva and a maintainer/developer of UserFrosting. I tend to focus on what I describe as the "assets pipeline". That is, the development server for static content and optimisation of said static content for production.

Looking ahead, I'm looking to evolve UserFrosting to make it more flexible when it comes to frontend with the goal being to support a few distinct "workloads" including;
* Separate API and GUI (e.g. UserFrosting API with React/Vue/Angular client app)
* Traditional MVC (e.g. UserFrosting v4 and lower)
* MVC with SPA (essentially enrichment of traditional MVC)

To support these variations, core UI components specific to UserFrosting would be created to handle in-box interactions (e.g. login, much of the admin panel, user settings). These components would be built for flexibility so as to avoid UI reimplementation in order to customise the look and feel.

Did I get side tracked there? Yes but also no, since it demonstrates who I am. A thinker and problem solver.
